vagrant/provisioning/roles/foia-analytical-reports/tasks/main.yml

```diff
@@ -0,0 +1,106 @@
+- name: create install folder
+  become: yes
+  become_user: pentaho
+  file:
+    path: "{{ root_folder }}/install/pentaho/foia-reports-dw-{{ foia_analytical_reports_version }}"
+    state: directory  
+
+- name: download FOIA analytical reports repository
+  become: yes
+  become_user: pentaho
+  command: sshpass -e sftp -o StrictHostKeyChecking\=no -o UserKnownHostsFile\=/dev/null {{ sftp_arkcase_user }}@{{ sftp_service_base_url }}:{{ sftp_arkcase_folder }}/{{ item }}
```

Comment:
@marjan4jc , we want to install the analytical reports on outside servers, like AWS, where EEOC production is hosted. But gitlab.armedia.com is locked up behind the Armedia VPN.  So, the sftp site is how we've arranged to make our artifacts available outside the VPN.

Reply:
@david-oc-miller, yes you are right, but maybe in the future, it wouldn't be better if we put the project on GitHub and use it from there, instead of some FTP server?

Reply:
Yes, no question about this.  Definitely better to move the repository to GitHub.